Backend-Challenges.md

```diff
@@ -48,6 +48,21 @@ Generally speaking, an NFT consists of following components:
 
 The goal of this challenge is to flesh out the constructor, have it accept the arguments passed in the test, and make it set the max supply inside the constructor to what was passed as an argument.
 
+### Solidity libraries and openzeppelin
+
+In Solidity, you can import libraries that will make life easier. OpenZeppelin is a company that provides free standardized contract libraries to use, such as ERC721. These contracts are firmly audited, meaning they are generally secure to implement in your contracts.
```

Comment:
If you want to stay that it makes life easier, i'd deepen a bit on the why: Something in the essence of: Because upgradability is hard, and the unforgiving environment for bugs plus time pressure for software development, libraries will make your life easier